_posts/2023-08-23-cockpit-299.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Cockpit 299
+author: tmatus
+date: '2023-08-23'
+tags: cockpit
+slug: cockpit-299
+category: release
+summary: ''
```

Comment:
This needs to be filled in.

Reply:
Ah, probably got lost when regenerating :)